src/angularjs/src/app/places/list/places-list.controller.js

```diff
@@ -14,6 +14,13 @@
                                  Neighborhood, AnalysisJob) {
         var ctl = this;
 
+        var sortingOptions = [
+            {value: 'neighborhood__label', label: 'Alphabetical'},
```

Comment:
@jcahail the scope changes doc states: "The examples on the wireframe are by default sorted by city in alphabetical order; CHANGE to sort alphabetically by state". So we sort first by state, then secondary sort by city name?